### DISCUSSION TITLE
Using Microsoft search with People entity and properties

### DISCUSSION THREAD

**Discussion by SinghGurd** [microsoft-search/pnp-modern-search#2226] [Q&A] [answered]
Is it possible to retrieve additional user profile properties when using Microsoft Search as a data source and People for entity? None of my changes for 'Selected Properties' are having any effect. Web part appear to only load some pre-defined set of fields.

**Comment by FranckyC** [comment, marked as answer]
Hi @SinghGurd, by additional properties you mean custom properties in the SharePoint user profile service? If this is the case, you can't do this with Microsoft Search as it work differently. The people search with Microsoft Search is basically a wrapper of  the [/people](https://docs.microsoft.com/en-us/graph/api/user-list-people?view=graph-rest-1.0&tabs=http) Graph API endpoint. It has nothing to do with SharePoint user profiles. If you need additional proeprties you may use SharePoint search data source instead.

**Reply by SinghGurd** [reply]
Thank you for the comment @FranckyC. I actually meant when trying to retrieve additional graph user properties such as businessPhones, mobilePhone. I am entering these in 'selected properties' field in web part property pane while using the Microsoft Search but it does not seem to have any effect.

**Reply by wobba** [reply]
@SinghGurd you should put that ask to the Graph API team who owns the Graph Search API over at https://docs.microsoft.com/en-us/answers/topics/microsoft-graph-search.html
